views/partials/header.hbs

```diff
@@ -1,18 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>
     {{#if publication}}
-     Publication: {{publication.title}} - Octopus
-    {{else}}
-      {{#if problem}}
-      Problem: {{problem.title}} - Octopus
-      {{else}}
-       Octopus. Built for Scientists.
-     {{/if}}
-   {{/if}}
+    Publication: {{publication.title}} - Octopus
```

Comment:
Can we turn this into a simpler server-side assignment?

Each controller can simply assign / overwrite `res.locals.customTitleTag` to provide the requested title as a string.

Then in template we can simplify this like this:
```
{{#if customTitleTag}}
      {{customTitleTag}} - Octopus
{{else}}
       Octopus. Built for Scientists.
{{/if}}
```

Reply:
@vot Done